core/src/avm2/vtable.rs

```diff
@@ -349,18 +386,23 @@ impl<'gc> VTable<'gc> {
                         Some(Property::Virtual {
                             set: Some(disp_id), ..
                         }) => {
+                            // Override metadata
```

Comment:
(IMO we don't really need this comment, the line below is already implied to be an overriding. We might wanna have a more generic comment just saying that this block corresponds to method overriding.)